tools/toitdoc/toitdoc.toit

```diff
@@ -193,15 +193,13 @@ compute-sdk-path --sdk-path/string? --toitc/string? --ui/Ui -> string:
     return sdk-path
 
   toitc = eval-symlinks toitc
-
-  // Try first with the directory of the toitc.
-  sdk-path = fs.dirname toitc
-  lib-dir := fs.join sdk-path "lib"
-  if file.is-directory lib-dir: return sdk-path
-  // Try "..".
-  sdk-path = fs.join sdk-path ".."
-  lib-dir = fs.join sdk-path "lib"
-  if file.is-directory lib-dir: return sdk-path
+  lib-bin-dir := fs.dirname toitc
```

Comment:
So this is the path to `toit.compile` which is in `lib/toit/bin/`, so you go back to `lib/toit/lib/`.

Reply:
Correct.